.github/workflows/test.yml

```diff
@@ -0,0 +1,18 @@
+on:
+  push:
+  pull_request:
+
+jobs:
+  eslint:
```

Comment:
is this for tests or linter? update these names if needed

Reply:
changed to be more specific to the test run